scripts/regenerate.sh

```diff
@@ -22,9 +22,9 @@ function finish {
 }
 trap finish EXIT
 
-export GOBIN=${WORKDIR}/bin
-export PATH=${GOBIN}:${PATH}
-mkdir -p ${GOBIN}
+export GOBIN="${WORKDIR}"/bin
+export PATH="${GOBIN}:${PATH}"
```

Comment:
Not part of your diff, but I still don't get why we need to export GOBIN here which is a `mktemp` which is also cleaned up in `trap`